### ISSUE TITLE
[Feature Request] get the anime data when searching anime.

### ISSUE THREAD

**Issue by itssxs989** [CTK-WARRIOR/ctk-anime-scraper#7]
The `search` function allows to get the title,  link, image and release date. It would be nice if you add a data object with these response. So the returning main object would include another object that includes other anime info like description, genre, type etc.
Thanks.

**Comment by LegendaryEmoji** [comment]
Hello, choppymaster 👋 

### TL;DR

- It will make package slow but we can add `scrapeFull` option to `search` options, so it will only scrape full when option available.

### Description

- Adding other properties like `description`, `genre`, `type`, `etc` is possible but it will make package slow

### Why (Fetching process)

For getting additional properties (`description`, `genre`, `etc`), scraper will have to fetch entire anime page, `search` method returns `url` property, scraper will have to send another `GET` request to anime `url` & then convert it to `textContent`, cheerio (npm pkg for making scraping work easier) will load all text content from anime `url` & get our additional properties, after that all it will make all raw content from cheerio into `textContent`.

### Solution / Idea

- Add `scrapeFull` property to `search` options, so it will only fetch additional properties when there is `scrapeFull` option available.

- Thanks

**Comment by itssxs989** [comment]
Good idea.

**Comment by ItzYuzuruYT** [comment]
Tru, the delay would really increase when loading multiple pages. You can instead use search and store the anime link in a var then use fetchAnime to fetch the data from that link. It'd be the same process
